.github/workflows/run_tests_python310_keras211.yml

```diff
@@ -1,14 +1,14 @@
-name: Run Tests - Tensorflow 2.12
+name: Python 3.10, Keras 2.11
 on:
   workflow_dispatch: # Allow manual triggers
-  schedule:
-    - cron: 0 0 * * *
   pull_request:
     branches:
       - main
 
+
 jobs:
-  run-tensorflow-2_12:
+  run-tests:
     uses: ./.github/workflows/run_keras_tests.yml
     with:
-      tf-version: "2.12.*"
+      python-version: "3.10"
```

Comment:
maybe use 3.10.* ? I'm not sure, but I remember that I encountered an issue when using 3.10 because sometimes it ignores the "0" and refers to it as "3.1"